src/Spryker/Client/RabbitMq/Model/Connection/ConnectionBuilder/ConnectionBuilder.php

```diff
@@ -114,14 +114,35 @@ protected function createAmqpStreamConnection(QueueConnectionTransfer $queueConn
             $queueConnectionTransfer->getLocale() ?? $this->getDefaultLocale(),
             $queueConnectionTransfer->getConnectionTimeout() ?? $defaultQueueConnectionTransfer->getConnectionTimeout(),
             $queueConnectionTransfer->getReadWriteTimeout() ?? $defaultQueueConnectionTransfer->getReadWriteTimeout(),
-            null,
+            $this->createSslContext($queueConnectionTransfer),
             $queueConnectionTransfer->getKeepAlive() ?? $defaultQueueConnectionTransfer->getKeepAlive(),
             $queueConnectionTransfer->getHeartBeat() ?? $defaultQueueConnectionTransfer->getHeartBeat(),
             $queueConnectionTransfer->getChannelRpcTimeout() ?? $defaultQueueConnectionTransfer->getChannelRpcTimeout(),
             $queueConnectionTransfer->getSslProtocol()
         );
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\QueueConnectionTransfer $queueConnectionTransfer
+     *
+     * @return resource|null
+     */
+    private function createSslContext(QueueConnectionTransfer $queueConnectionTransfer)
```

Comment:
why this is private? We can make it protected if it's not required in this case to be private

Reply:
I thought private was better in this scenario cause this logic shouldn't be overwritten.

But if you see a scenario where it actually could, I can make it protected.